content/en/building/branding/application-graphics.md

```diff
@@ -8,8 +8,8 @@ relatedContent: >
   building/admin
   design/interface/icons
 aliases:
-   - /apps/tutorials/application-graphics
-   - /building/tutorials/application-graphics
+  - /apps/tutorials/application-graphics
+  - /building/tutorials/application-graphics
```

Comment:
while git easily allows you to ignore white space, a best practice to not change a file's spacing unless you're changing the content itself.  Not a big deal, just keep it in mind for next time!